src/config/logtest-config.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (C) 2015-2020, Wazuh Inc.
+ * Copyright (C) 2009 Trend Micro Inc.
+ * All right reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
+#include "shared.h"
+
+/* Internal options default values */
+#define LOGTEST_THREAD                  1
+#define LOGTEST_LIMIT_THREAD            128
+#define LOGTEST_MAX_SESSIONS            64
+#define LOGTEST_LIMIT_MAX_SESSIONS      500
+#define LOGTEST_SESSION_TIMEOUT         900
+#define LOGTEST_LIMIT_SESSION_TIMEOUT   31536000
+
+/**
+ * @brief Struct to save the wazuh-logtest internal configuration
+ */
+typedef struct w_logtest_config_t {
+
+    char *enabled;
```

Comment:
I think `enabled` should be a boolean. Is there any reason to keep it being a string?